R/pkg/R/mllib.R

```diff
@@ -192,3 +210,37 @@ setMethod("fitted", signature(object = "PipelineModel"),
               stop(paste("Unsupported model", modelName, sep = " "))
             }
           })
+
+#' Fit a naive Bayes model
+#'
+#' Fit a naive Bayes model, similarly to R's naiveBayes() except for omitting two arguments 'subset'
+#' and 'na.action'. Users can use 'subset' function and 'fillna' or 'na.omit' function of DataFrame,
+#' respectviely, to preprocess their DataFrame. We use na.omit in this interface to avoid potential
+#' errors.
```

Comment:
MLlib NaiveBayes cannot handle NA values, so I need to remove them with `na.omit`.